_docs/conversions.md

```diff
@@ -0,0 +1,180 @@
+
+    Learn how to create a token on Quay.io for your organizations here under the heading "Generating a Token (for internal application use)". You will need to provide these permissions:
+    
+    * Create Repositories
+    
+    * View all visible repositories
+    
+    * Read/Write to any accessible repository
+    
+#### Server Configuration
+
+The web service alone only requires a GitHub and Quay.io token. There are two ways to specify your tokens.
+
+1. Environmental variables.
+    You can set them in Bash like the following:
+```
+export quayioToken=<your token here>
+export githubToken=<your token here>
+```
+2. The YAML configuration file. The tokens can be entered in the top two lines. An example of a YAML configuration file can be seen here.
+
+Run the service using a configuration file:
+`java -jar write-api-service-*.jar server example.yml`
+The example.yml shown previously uses port 8082 by default, this can be changed. Note this port number, it will later be used for the Write API Client properties file.
+
+After running the webservice, you can check out the web service endpoints through swagger. By default, it is available at http://localhost:8082/static/swagger-ui/index.html.
+
+The basic workflow is that GitHub repos are created when posting a new tool. When files are posted or put to a version of a tool, we will create or delete and re-create a GitHub release/branch/tag with a matching name. When Dockerfiles are added, the tool will be created and built as a Quay.io repo. After adding both Dockerfiles and descriptors, you basically have a tool that is ready to be quickly registered and published under a Dockstore 1.2 web service. Go to Dockstore, do a refresh, and then hit quick register on the repos that you wish to publish. You can also do this programmatically through the write api client
+
+#### Client Prerequisites
+
+* Write API web service and all its prerequisites
+
+By now, then web service should be up and running with valid GitHub and Quay.io tokens. If not, please return to the web service usage section to get that running first. It is advised to ensure that the Write API web service is functioning correctly before using the client.
+
+* [Dockstore token](https://docs.dockstore.org/docs/publisher-tutorials/register-on-dockstore/)
+
+Follow the "Getting Started with Dockstore" tutorial to get a Dockstore token. Note this down, it will later be used in the Write API client properties file.
+
+* Dockstore server-url
+
+The Dockstore tutorial earlier would've specified the server-url alongside the token. Unless you're running your own dockstore webservice, the Dockstore production server-url is "https://www.dockstore.org:8443" and the Dockstore staging server-url is "https://staging.dockstore.org:8443". Note this down, it will also later be used in the Write API client properties file.
```

Comment:
Remove port and add `api`: https://dockstore.org/api. I don't think we mention the staging instance anywhere else in the docs, so I would remove that reference.